scripts/linters/custom_eslint_checks/rules/e2e-action-exclude.json

```diff
@@ -0,0 +1,62 @@
+{
+    "exclude": [
+        "ClassroomPage.js",
```

Comment:
We are trying to avoid having oppia project-specific things/string in the linter project. Can you please check whether we can pass these configs through `.eslintrc`?

Reply:
done